### PR TITLE
Added networking stack main vpc security groups

### PR DESCRIPTION
**Pull request by victorskl**
* Added outbound only sg
* Added outbound and uom inbound only to ssh sg

### CODE REVIEW

File: terraform/stacks/networking/main.tf [hunk 1]
```diff
@@ -103,3 +103,52 @@ module "main_vpc" {
...
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name        = "outbound"
```
Comment (reisingerf) [inline terraform/stacks/networking/main.tf:120]: I'd use outbound_only, just to be clear

File: terraform/stacks/networking/main.tf [hunk 2]
```diff
@@ -103,3 +103,52 @@ module "main_vpc" {
...
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["10.1.0.0/20"]
+  }
+
+  tags = {
+    Name        = "uom"
```
Comment (reisingerf) [inline terraform/stacks/networking/main.tf:148]: I'd use ssh_from_uom, again to be clearer

File: terraform/stacks/networking/main.tf (hunk 1 above)
Comment (victorskl) [reply terraform/stacks/networking/main.tf]: Amended

File: terraform/stacks/networking/main.tf (hunk 2 above)
Comment (victorskl) [reply terraform/stacks/networking/main.tf]: Amended

**Pull request merged**